src/map/map.ts

```diff
@@ -630,7 +846,34 @@ export class PropertiesMap {
             showlegend: false,
         };
 
-        const traces = [main as Data, selected as Data];
+        // The main trace, containing default data
```

Comment:
Shouldn't this comment read "the background trace, containing transparent data"?

Reply:
Fixed!